### PR TITLE
Jg favorite indicator us8

### PR DESCRIPTION
**Pull request by elguapogordo**
add test for favorite pets link in nav bar for the following pages
  * /pets/index
  * /pets/show
  * /pets/new
  * /reviews/new
  * /reviews/edit
  * /shelterpets/index
  * /shelterpets/new
  * /shelters/index
  * /shelters/new
  * /shelters/show
  * /shelters/edit

add link to nav bar

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -15,6 +15,7 @@
     <nav>
       <li><%= link_to "Shelters Index", "/" %></li>
       <li><%= link_to "Pets Index", "/pets" %></li>
+      <li><%= link_to "Favorite Pets", "/favorites" %></li>
```
Comment (elguapogordo) [inline app/views/layouts/application.html.erb]: 95 lines of test code to test this one line

**Comment by elguapogordo** [comment]
the other files were from the code tony pushed that was in master but not dev branch at the time i began work on this.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (tony-gomes) [reply app/views/layouts/application.html.erb]: I think @mikez321 said he was going to inquire about a better approach to this, but you are trooper for getting it done...

File: spec/features/favorites/favorite_indicator_spec.rb [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+     review_image: "https://via.placeholder.com/150",
+     title: "test review 1",
+     rating: 1,
+     content: "this review is average at best."
+   )
+
+    visit "/shelters"
+
+    within "nav" do
+      expect(page).to have_link("Favorite Pets")
```
Comment (tony-gomes) [inline spec/features/favorites/favorite_indicator_spec.rb]: Great use of the semantic tag @elguapogordo! However difficult it is to say with your handle...

File: spec/features/shelterreviews/shelter_review_edit_cont_spec.rb [hunk 3]
```diff
@@ -20,8 +20,7 @@
 
       visit "/shelters/#{@shelter_1.id}/reviews/#{@review_1.id}/edit"
 
-      fill_in 'title', with: "review title"
-      fill_in 'rating', with: 5
+      fill_in 'content', with: ""
```
Comment (tony-gomes) [inline spec/features/shelterreviews/shelter_review_edit_cont_spec.rb]: Great edge case @mikez321 & @elguapogordo...

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (mikez321) [reply app/views/layouts/application.html.erb]: Yeah, there has got to be a better way... props to you for this test!

File: spec/features/favorites/favorite_indicator_spec.rb [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+   review_1 = shelter_1.reviews.create!(
+     review_image: "https://via.placeholder.com/150",
+     title: "test review 1",
+     rating: 1,
+     content: "this review is average at best."
+   )
+
+    visit "/shelters"
+
+    within "nav" do
```
Comment (mikez321) [inline spec/features/favorites/favorite_indicator_spec.rb]: You guys were right about testing for this, I like how we have it narrowed down to the nav bar.  Good call.

**Pull request merged**